infrastructure/templates/test-runner/image-template.json

```diff
@@ -0,0 +1,74 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "imageTemplateName": {
+      "type": "string"
+    },
+    "svcLocation": {
+      "type": "string"
+    },
+    "managedImageResGroupId": {
+      "type": "string"
+    },
+    "managedImageName": {
+      "type": "string"
+    }
+  },
+  "variables": {
+    "resourceId": "[concat(parameters('managedImageResGroupId'),parameters('managedImageName'))]"
+  },
+  "resources": [
+    {
+      "type": "Microsoft.VirtualMachineImages/imageTemplates",
+      "name": "[parameters('imageTemplateName')]"
```

Comment:
Seems like we're missing an `apiVersion` property. We should set this to `2021-10-01` according to the docs